Documentation/signing-and-verification-guide.md

```diff
@@ -0,0 +1,333 @@
+```
+
+#### Capture the public key fingerprint
+
+```
+$ gpg --no-default-keyring --fingerprint --keyring ./pubkeys.gpg kelsey.hightower@coreos.com
+```
+
+```
+pub   2048R/26EF7A14 2015-01-09
+      Key fingerprint = B346 E31D E7E3 C6F9 D1D4  603F 4DFB 61BF 26EF 7A14
+uid       [ unknown] Kelsey Hightower (ACI signing key) <kelsey.hightower@coreos.com>
+sub   2048R/B9C074CD 2015-01-09
+```
+
+Remove white spaces and convert to lowercase:
+
+```
+$ echo "B346 E31D E7E3 C6F9 D1D4  603F 4DFB 61BF 26EF 7A14" | \
+  tr -d "[:space:]" | tr '[:upper:][:lower:]' '[:lower:][:upper:]'
+```
+```
+b346e31de7e3c6f9d1d4603f4dfb61bf26ef7a14
+```
+
+#### Trust the key for the example.com/hello prefix
+
+```
+mkdir -p /etc/rkt/trustedkeys/prefix.d/example.com/hello
+mv pubkeys.gpg  /etc/rkt/trustedkeys/prefix.d/example.com/hello/b346e31de7e3c6f9d1d4603f4dfb61bf26ef7a14
+```
+
+Now the public key with fingerprint `b346e31de7e3c6f9d1d4603f4dfb61bf26ef7a14` will be trusted for all images with a name prefix of `example.com/hello`.
+If you would like to trust a public key for any image store the public key in one of the following directories:
+
+```
+/etc/rkt/trustedkeys/root.d
+/usr/lib/rkt/trustedkeys/root.d
+```
+
+### Example Usage
+
+```
+$ sudo rkt -h
+Usage of rkt:
+  -debug=false: Print out more debug information to stderr
+  -dir="/var/lib/rkt": rocket data directory
+  -help=false: Print usage information and exit
+  -insecure-skip-verify=false: skip image verification
+```
+
+#### Download, verify and run an ACI
+
+By default rocket will attempt to download the ACI detached signature and verify the image:
+
+```
+$ sudo rkt run example.com/hello:0.0.1
+rkt: starting to discover app img example.com/hello:0.0.1
+rkt: starting to fetch img from http://example.com/images/example.com/hello-0.0.1-linux-amd64.aci
+Downloading aci: [                                             ] 7.24 KB/1.26 MB
+rkt: example.com/hello:0.0.1 verified signed by:
+  Kelsey Hightower (ACI signing key) <kelsey.hightower@coreos.com>
+/etc/localtime is not a symlink, not updating container timezone.
+^]^]Container stage1 terminated by signal KILL.
+```
+
+Use the `-insecure-skip-verify` flag to disable image verification for a single run:
+
+```
+$ sudo rkt -insecure-skip-verify run example.com/hello:0.0.1
+rkt: starting to discover app img example.com/hello:0.0.1
+rkt: starting to fetch img from http://example.com/images/example.com/hello-0.0.1-linux-amd64.aci
+rkt: warning signature verification has been disabled
+Downloading aci: [=                                            ] 32.8 KB/1.26 MB
+/etc/localtime is not a symlink, not updating container timezone.
+^]^]Container stage1 terminated by signal KILL.
+```
+
+Notice when the `-insecure-skip-verify` flag is used, rocket will print the following warning:
+
+```
+rkt: warning signature verification has been disabled
+```
+
+#### Download and verify an ACI
+
+Using the fetch subcommand you can download and verify an ACI:
```

Comment:
"without immediately running a container. This can be useful to precache ..." or somesuch